Content.Server/Administration/Commands/RoleBanCommand.cs

```diff
@@ -22,6 +25,7 @@ public async void Execute(IConsoleShell shell, string argStr, string[] args)
         string job;
         string reason;
         uint minutes;
+        var severity = NoteSeverity.Minor;
```

Comment:
That's completely fair, but it might be better to seek opinions of the curent admin team.

I've reworked the ban panel to include role bans, and hopefully people don't use the department ban command anymore.

Reply:
How I've seen it handled most of the time is by feeling. Both notes and bans have all kinds of severities, depending on the nature of what transpired. Minor for warnings and short bans, medium for more severe cases, and high for inexcusable severe lapses of judgement

Reply:
From my experience subjective judgements are useful enough, without creating policy overhead. They help identify more severe bans at a glance. You need to read someone's note history either way, as taking an unrelated note into account wouldn't be fair. Maybe it's just me. It's out of scope for this PR, I'll be adding config values so everyone can configure them to their liking.